docs/sphinx/source/reference/sql_commands/DQL/WITH.diagram

```diff
@@ -1,29 +1,49 @@
 Diagram(
     Stack(
         Terminal('WITH'),
-            OneOrMore(
-                Sequence(
-                    NonTerminal('cteName'),
-                    Optional(
-                        Sequence(
-                            Terminal('('),
-                            OneOrMore(
-                                NonTerminal('columnAlias'),
-                                Terminal(','),
-                            ),
-                            Terminal(')'),
+        Optional(
+            Sequence(
+                Terminal('RECURSIVE'),
+                Optional(
+                    Sequence(
+                        Terminal('('),
+                        Terminal('TRAVERSAL'),
+                        Terminal('='),
+                        Choice(0,
+                            Terminal('ANY'),
```

Comment:
Do we need an `ANY`, or could we assume that the absence of a specified traversal order is `ANY`, a little like the absence of an index hint means any index is fair game.